web3.js/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
it might be worth removing babel/rollup, and have two typescript files similar to https://github.com/solana-labs/token-list

Reply:
Yeah that would be quite nice, unfortunately we need node polyfills for the browser and afaik typescript's bundler doesn't support that